documentation/RLC-quickstart.md

```diff
@@ -85,22 +110,14 @@ We are working on to automatically generate everything right now, but currently
 
     --- 
 
-4. **Run command to generate the SDK**  
-
-    We need to configure `--output-dir` to put generated code. The output dir contains two parts: the {SDK_REPO_ROOT} and {PROJECT_ROOT}.
-    
-    Assume **{SDK_REPO_ROOT}** is `D:/azure-sdk-for-js` and **{PROJECT_ROOT}** is `sdk/agrifood/agrifood-farming-rest` then we could run this command in **your local TypeSpec project** to generate the SDK: 
-
-    ```shell
-    npx tsp compile . --emit=@azure-tools/typespec-ts --output-dir=D:/azure-sdk-for-js/sdk/agrifood/agrifood-farming-rest
-    ```
+4. **Build your project**  
```

Comment:
Same comment as above, lets have this in a separate doc or if possible linking to the Contributing.md doc

Reply:
move to after generation doc